env_config.rb

```diff
@@ -19,6 +19,7 @@
     mandatory :DATABASE_AWS_REGION, :string
     mandatory :DATABASE_WRT_USER, :string
     optional :PAYPAL_BASE_URL, :string ## TODO: Change to mandatory when launching paypal
+    optional :WRC_WEBHOOK_URL, :string, ''
```

Comment:
Not sure if I'm happy with `optional` in the prod context, we usually want the startup to crash (and thus fail the B/G deployment, causing no apparent harm to the Average Joe website user) when we forgot to configure something